packages/terra-show-hide/src/_ShowHideButton.jsx

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable react/button-has-type */
```

Comment:
Reviewers, this is essentially a copy/paste of terra-button's code so we get all the functionality we have in terra-button but with a "link style" applied to the button. We went this approach as UX doesn't want the link style button to be highly used like the other button variants.